<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tareas</class>
 <widget class="QWidget" name="tareas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>ToDo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(222, 221, 218);
border-top-color: rgb(36, 31, 49);
gridline-color: rgb(36, 31, 49);</string>
  </property>
  <widget class="QLabel" name="Encabezado">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Lista de Tareas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_agregar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>15</pointsize>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(143, 240, 164);

</string>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_modificar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>15</pointsize>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(143, 240, 164);

</string>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boton_eliminar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>15</pointsize>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(143, 240, 164);
</string>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QLabel" name="pregunta_quehacer">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>¿Que hay que hacer?</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="ingreso_texto_tarea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
    border-radius: 10px;
    border: 0.5px solid gray;
padding: 5px;
}
</string>
   </property>
   <property name="placeholderText">
    <string>Ingrese aquí la tarea</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="area_de_lista">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
    border-radius: 10px;
    border: 0.5px solid gray;
padding: 5px;
background-color: rgb(46, 194, 126);
}</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>319</width>
      <height>219</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(153, 193, 241);</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="boton_completada">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Light</family>
     <pointsize>15</pointsize>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(143, 240, 164);

</string>
   </property>
   <property name="text">
    <string>Completada</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo_imagen">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/cct/descarga.png);
border -radius: 20px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ingreso_texto_tarea</tabstop>
  <tabstop>boton_agregar</tabstop>
  <tabstop>boton_modificar</tabstop>
  <tabstop>boton_completada</tabstop>
  <tabstop>boton_eliminar</tabstop>
  <tabstop>area_de_lista</tabstop>
 </tabstops>
 <resources>
  <include location="logotareas.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>boton_agregar</sender>
   <signal>clicked()</signal>
   <receiver>tareas</receiver>
   <slot>boton_agregar_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_modificar</sender>
   <signal>clicked()</signal>
   <receiver>tareas</receiver>
   <slot>boton_modificar_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_completada</sender>
   <signal>clicked()</signal>
   <receiver>tareas</receiver>
   <slot>boton_completar_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_eliminar</sender>
   <signal>clicked()</signal>
   <receiver>tareas</receiver>
   <slot>boton_eliminar_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>boton_agregar_clicked()</slot>
  <slot>boton_modificar_clicked()</slot>
  <slot>boton_completar_clicked()</slot>
  <slot>boton_eliminar_clicked()</slot>
 </slots>
</ui>
